style.css

```diff
@@ -592,6 +739,85 @@ h3 {
 /* Properties for larger screens */
 
 @media (min-width: 768px) {
+  .overlay {
+    position: fixed;
+    top: 0;
+    left: 0;
+    right: 0;
+    height: 100vh;
+    width: 100%;
+    overflow: auto;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    background: rgba(193, 199, 208, 0.5);
+    z-index: 200;
+    backdrop-filter: blur(5px);
+  }
+
+  .modal-content {
+    margin: 5rem 3.5rem 0 3.5rem;
+  }
+

```

Comment:
- [ ] Good job adding the js linters checks as required by the previous reviewer, :clap: 
Kindly make sure you stick to the design by respecting the proportion of the size of the of the image and it's margins. on both desktop and mobile version.

- [ ] Again like the previous reviewer suggested in his illustration please make sure you fix the buttons designs to meet the Figma design, and also adding the close button on mobile as it is not showing up.

## Your mobile version now:

- mobile
![yourMob](https://user-images.githubusercontent.com/97834160/209412997-7d780ed1-18e7-401d-9152-c3d35fe1ffd2.png)
- desktop

![youDesk](https://user-images.githubusercontent.com/97834160/209412999-0f5e438b-d88d-40bd-8cc1-b634635cc2bb.png)

Figma designs:
- mobile
![figMob](https://user-images.githubusercontent.com/97834160/209413053-e3799863-b16f-46fe-af71-1e2db41f2da5.png)
- desktop
![figDesk](https://user-images.githubusercontent.com/97834160/209413060-723fb6fe-7435-4891-bd76-aa8e4432d362.png)